db/seeds.rb

```diff
@@ -5,3 +5,39 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+require 'faker'
+
+25.times do
+  user = User.new(
+    first_name: Faker::Name.first_name,
+    last_name: Faker::Name.last_name ,
+    email: Faker::Internet.email,
+    password: 'secret'
+  )
+  user.save!
+end
+
+12.times.with_index do |index|
+  castle_image = File.open(Rails.root.join("db/fixtures/castles/castle#{index + 1}.jpg"))
+  castle = Castle.new(
+    name:  Faker::Games::Pokemon.name,
+    description: "description",
+    address: Faker::Address.city,
+    price_per_day: rand(10..20)
+  )
+  castle.photo.attach(io: castle_image, filename: "castle#{index + 1}.jpg")
+  random_user = User.order(Arel.sql('RANDOM()')).first
+  castle.user = random_user
+  castle.save!
+end
+
+
+# User.destroy_all
```

Comment:
you can put these line at the top without comments
+ met un petit "puts "creating database"
+ test la validité des User et des Castle avec user.valid? et castle.valid?